kitsune/sumo/static/js/gallery.js

```diff
@@ -309,7 +318,7 @@
                   .html(message);
         },
         /*
-         * Fired if isValidFile is false or server returned failure.
+         * Fired if isValidFile or isNotTooLarge is false or server returned failure.
```

Comment:
Same here, `isNotTooLarge` is an implementation details of `isValidFile`, not something that needs documented on this function.